ethereum_history_api/circuits/lib/src/verifiers/account.nr

```diff
@@ -19,12 +18,10 @@ pub(crate) fn assert_account_equals(account_rlp_left_padded: [u8; MAX_ACCOUNT_ST
     let account_rlp_list: RlpList<ACCOUNT_FIELDS_COUNT> = decode_list_of_small_strings(account_rlp_right_padded);
     assert(account_rlp_list.len == ACCOUNT_FIELDS_COUNT, "Invalid number of fields in account RLP");
 
-    let nonce = u64_to_u8(account.nonce as u64);
-    let nonce = byte_value(nonce);
     let balance = U128::from_integer(account.balance).to_be_bytes();
     let balance = byte_value(balance);
 
-    account_rlp_list.get(NONCE_INDEX).assert_string_eq("Nonce", account_rlp_right_padded, nonce);
+    account_rlp_list.get(NONCE_INDEX).assert_u64_eq("Nonce", account_rlp_right_padded, account.nonce as u64);
```

Comment:
I know it is a work in progress, but I didn't see it on the list, so I would propose to have: `assert_field_eq`, so that we don't have to cast:
```
assert_field_eq("Nonce", account_rlp_right_padded, account.nonce as field)
```

Reply:
We don't want to add this function, because we will change all of the Field values to different types in the next PR

Reply:
Ahhh, good point 👍